documentation/asciidoc/topics/proc_creating_sa_tokens.adoc

```diff
@@ -0,0 +1,63 @@
+[id='creating_sa_tokens-{context}']
+= Creating Service Account Tokens
+{ocp} clusters use service account tokens to authenticate with each other.
+
+.Procedure
+
+. Create a service account token.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc create sa site-a
+serviceaccount/site-a created
+----
++
+. Get the secrets from your {openshiftshort} namespace, for example:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc get secrets
+
+NAME                       TYPE
+...
+site-a-token-8j6d8         kubernetes.io/service-account-token
+site-a-token-g2brv         kubernetes.io/service-account-token
+----
++
+. Retrieve the token from one of the secrets.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc describe secret site-a-token-8j6d8
+
+Name:         site-a-token-8j6d8
+Namespace:    local-operators
+Labels:       <none>
+Annotations:  kubernetes.io/created-by: openshift.io/create-dockercfg-secrets
+              kubernetes.io/service-account.name: site-a
+              kubernetes.io/service-account.uid: c22bc8d6-ffd8-11e9-adb6-0a8e23c49339
+
+Type:  kubernetes.io/service-account-token
+
+Data
+====
+ca.crt:          4634 bytes
+namespace:       15 bytes
+service-ca.crt:  5750 bytes
+token:           eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9...
+----
++
+. Add the service account tokens to secrets on each site so it can authenticate with the backup site.
++
+For example, in the preceding example you created a service account token for SiteA. You should add that token to a secret on SiteB. Doing so allows SiteB to authenticate with SiteA.
++
+.. Log in to {ocp} for SiteB.
+.. Add the service account token for SiteA to a secret.
++
+[source,options="nowrap",subs=attributes+]
+----
+oc create secret generic site-a-token --from-literal=token=eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9...
```

Comment:
In the minikube example you create secrets for site-b. I think both examples should refer to same site for consistency?